File: composeApp/src/desktopMain/kotlin/com/katsukosail/soundbox/database/model/Genero.java
package com.katsukosail.soundbox.database.model;

public class Genero {
    private int idGenero;
    private String descripcion;

    public Genero(int idGenero, String descripcion) {
        this.idGenero = idGenero;
        this.descripcion = descripcion;
    }

    public Genero(String descripcion) {
        this.descripcion = descripcion;
    }

    public int getIdGenero() {
        return idGenero;
    }
    public void setIdGenero(int idGenero) {
        this.idGenero = idGenero;
    }
    public String getDescripcion() {
        return descripcion;
    }
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    @Override
    public String toString() {
        return "Genero{" +
                "idGenero=" + idGenero +
                ", descripcion='" + descripcion + '\'' +
                '}';
    }
}

File: composeApp/src/desktopMain/kotlin/com/katsukosail/soundbox/database/model/Genero.kt
package com.katsukosail.soundbox.database.model

class Genero {
    var idGenero: Int = 0
    var descripcion: String?

    constructor(idGenero: Int, descripcion: String?) {
        this.idGenero = idGenero
        this.descripcion = descripcion
    }

    constructor(descripcion: String?) {
        this.descripcion = descripcion
    }

    override fun toString(): String {
        return "Genero{" +
                "idGenero=" + idGenero +
                ", descripcion='" + descripcion + '\'' +
                '}'
    }
}
